enterprise/backend/test/metabase_enterprise/serialization/v2/load_test.clj

```diff
@@ -940,3 +939,97 @@
                        :template-tags
                        (get "snippet: things")
                        :snippet-id)))))))))
+
+(deftest card-has-parameter-with-source-card-test
```

Comment:
There's no tests I can see for correct serdes with a literal value list. Is that a separate PR? I don't see any reason why it shouldn't Just Work, but we should test it.

Reply:
Also consider adding this new kind of linking to the test data generation used by the e2e test.

That system is kind of weird, I'm happy to point the way, or pair on it.